distribution/sender/routes/database-event-reader.xml

```diff
@@ -0,0 +1,115 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+
+    <route id="db-event-reader">
+        <from uri="scheduler:retry?initialDelay=1000&amp;delay=10000" />
+  
+		<log message="Fetching events in the staging table" loggingLevel="DEBUG" />
+    
+        <toD uri="jpa:DebeziumEvent?query=SELECT dE.id FROM DebeziumEvent dE ORDER BY event_time_stamp ASC" />
+			
+			
+       	 <choice>
+            <when>
+                <simple>${body.size()} > 0</simple>
+                <log message="Event count in the stating area: ${body.size()}" />
+                 
+                 <split parallelProcessing="false" stopOnException="true">
+                 	<simple>${body}</simple>
+                 	
+                	<log message="Loading event item with id: ${body}" loggingLevel="INFO"/>
```

Comment:
INFO is the default log level, so you can remove the attribute

Reply:
Ok